guide.md

```diff
@@ -195,13 +195,14 @@ While working on this module, you will learn how to access constants and variabl
 
 You may display this via a hook or a project page.
 
-The goal of this exercise is to create a module that displays a user's:
-1. Username
-1. Admin (aka superuser) status
-1. Their user rights
-1. The current page path
-1. The current project's `project_id`
-1. The value of a variable set in the module's configuration menu
+The goal of this exercise is to create a module that:
+1. Displays Username
+2. Displays Admin (aka superuser) status
+3. Displays Their user rights
+4. Displays The current page path
+5. Displays The current project's `project_id`
+6. Creates a custom variable on the project's configuration menu
```

Comment:
When I was checking the answers after doing this module, I noticed I had done this wrong because I had pulled an existing module setting into the display, when the correct answer actually had a custom variable set up.

Reply:
Ah! I did NOT know that, I love it. thanks for explaining.